application/Dockerfile

```diff
@@ -2,11 +2,16 @@ FROM openjdk:8-jre-slim
 
 RUN \
       addgroup --system franklin \
-      && adduser --system --disabled-login --disabled-password --home /var/lib/franklin --shell /sbin/nologin --ingroup franklin franklin
+      && adduser --system --disabled-login --disabled-password \
+      --home /var/lib/franklin \
+      --shell /sbin/nologin \
+      --ingroup franklin \
+      -u 1000 \
```

Comment:
What problem did this solve?

Reply:
If you don't set the UID, then when you run the container locally and mount in AWS creds, the user in the container isn't allowed to read them.

Reply:
...which caused a bunch of access errors when trying to read from AWS